src/utils/common.js

```diff
@@ -0,0 +1,28 @@
+/*  
+    classNames example usage:
+    <div className={classNames(['base-class', {'second' : someFlag, 'third' : true}])}></div>
+*/
+const classNames = (names) => {
+    let isArray = Array.isArray;
+    let i;
```

Comment:
it's best to declare variables close to where they are used and in most limited scope, i.e. in this case `i` and `len` should be declared in `for` loop.